src/Service.Tests/SqlTests/GraphQLSupportedTypesTests/GraphQLSupportedTypesTestsBase.cs

```diff
@@ -289,19 +277,25 @@ public async Task TestTimeTypePrecisionCheck(string gqlValue, int count)
         [DataRow(BOOLEAN_TYPE, "true")]
         [DataRow(BOOLEAN_TYPE, "false")]
         [DataRow(BOOLEAN_TYPE, "null")]
-        [DataRow(DATETIMEOFFSET_TYPE, "\"1999-01-08 10:23:54+8:00\"")]
-        [DataRow(DATETIMEOFFSET_TYPE, "\"1999-01-08 10:23:54.671287+8:00\"")]
-        [DataRow(DATETIME_TYPE, "\"1999-01-08 09:20:00\"")]
-        [DataRow(DATETIME_TYPE, "\"1999-01-08\"")]
-        [DataRow(DATETIME_TYPE, "null")]
         [DataRow(BYTEARRAY_TYPE, "\"U3RyaW5neQ==\"")]
+        [DataRow(BYTEARRAY_TYPE, "\"V2hhdGNodSBkb2luZyBkZWNvZGluZyBvdXIgdGVzdCBiYXNlNjQgc3RyaW5ncz8=\"")]
+        [DataRow(BYTEARRAY_TYPE, "null")]
         [DataRow(TIME_TYPE, "\"23:59:59.9999999\"")]
         [DataRow(TIME_TYPE, "\"23:59:59\"")]
         [DataRow(TIME_TYPE, "\"23:59:59.9\"")]
         [DataRow(TIME_TYPE, "\"23:59\"")]
         [DataRow(TIME_TYPE, "null")]
-        [DataRow(BYTEARRAY_TYPE, "\"V2hhdGNodSBkb2luZyBkZWNvZGluZyBvdXIgdGVzdCBiYXNlNjQgc3RyaW5ncz8=\"")]
-        [DataRow(BYTEARRAY_TYPE, "null")]
+        [DataRow(DATETIME_TYPE, "\"1999-01-08 09:20:00\"")]
+        [DataRow(DATETIME_TYPE, "\"1999-01-08\"")]
```

Comment:
Is there a test for max value of DATETIME type insertion?

Reply:
added